{"name": "Validate API response for negative limit", "status": "failed", "statusDetails": {"message": "AssertionError: Expected 400, but got 404\n", "trace": "  File \"c:\\Projects\\CatFactsQe\\.venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\Projects\\CatFactsQe\\.venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\cat_api_steps.py\", line 13, in step_check_status_code\n    assert context.response.status_code == status_code, \\\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "steps": [{"name": "Given I send a GET request to \"/breeds?limit=-1\"", "status": "passed", "start": 1740359675186, "stop": 1740359675434}, {"name": "Then the response status should be 400", "status": "failed", "statusDetails": {"message": "AssertionError: Expected 400, but got 404\n", "trace": "  File \"c:\\Projects\\CatFactsQe\\.venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\Projects\\CatFactsQe\\.venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\cat_api_steps.py\", line 13, in step_check_status_code\n    assert context.response.status_code == status_code, \\\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1740359675434, "stop": 1740359675447}], "start": 1740359675185, "stop": 1740359675447, "uuid": "b5219b71-bf95-491b-88c8-7ef986ab0bfe", "historyId": "b8674184788e82a300c6bb32187699ac", "fullName": "Validate Cat Breeds API: Validate API response for negative limit", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Validate Cat Breeds API"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}